doc/mgr/dashboard.rst

```diff
@@ -287,7 +287,17 @@ More details can be found on the `documentation <http://docs.ceph.com/docs/maste
         grafana-cli plugins install vonage-status-panel
         grafana-cli plugins install grafana-piechart-panel
 
-#. Add the Dashboards to Grafana by importing them
+#. Add the Dashboards to Grafana:
+
+   Dashboards can be added to Grafana by importing dashboard jsons. Following command can be used for downloading json files::
```

Comment:
Please try to keep the lines to 80 cols.
Code blocks are not that easy to break, so you can keep it like that.